content/en/docs/collector/troubleshooting.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Troubleshooting
+weight: 32
+---
+
+The OpenTelemetry Collector supports a variety of metrics, logs, and extensions
+for troubleshooting collector health and performance.
+
+Detailed recommendations, including common problems, are detailed in the
+OpenTelemetry Collector GitHub repo's
+[troubleshooting document](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md).
```

Comment:
I'd suggest moving the content of this paragraph (lines 9 to 11) to the end, under a section entitled (something like) **More**.